src/main/resources/db/migration/V7__add_address_contact_defendant_alias_tables.sql

```diff
@@ -0,0 +1,94 @@
+BEGIN;
+-------- address -------------------------------------------------------
+DROP TABLE IF EXISTS address;
+
+CREATE TABLE IF NOT EXISTS address
+(
+    id                                      SERIAL      PRIMARY KEY,
+    address_line_one                        TEXT        DEFAULT NULL,
+    address_line_two                        TEXT        DEFAULT NULL,
+    address_line_three                      TEXT        DEFAULT NULL,
+    address_line_four                       TEXT        DEFAULT NULL,
+    address_line_five                       TEXT        DEFAULT NULL,
+    postcode                                TEXT        DEFAULT NULL,
+    version                                 int4        NOT NULL DEFAULT 0
+
+);
+-------------------------------------------------------------------
+
+-------- contact -----------------------------------------------------
+DROP TABLE IF EXISTS contact;
+
+CREATE TABLE IF NOT EXISTS contact
+(
+    id                                      SERIAL      PRIMARY KEY,
+    home_phone                              TEXT        DEFAULT NULL,
+    work_phone                              TEXT        DEFAULT NULL,
+    mobile                                  TEXT        DEFAULT NULL,
+    primary_email                           TEXT        DEFAULT NULL,
+    version                                 int4        NOT NULL DEFAULT 0
+);
+---------------------------------------------------------------------
+
+--------- defendant_alias ------------------
+DROP TABLE IF EXISTS defendant_alias;
+
+CREATE TABLE IF NOT EXISTS defendant_alias
+(
+    id                                      SERIAL      PRIMARY KEY,
+    first_name                              TEXT        DEFAULT NULL,
+    middle_name                             TEXT        DEFAULT NULL,
+    surname                                 TEXT        DEFAULT NULL,
+    version                                 int4        NOT NULL DEFAULT 0,
```

Comment:
Do we have to handle this?
It looks like it used to be part of BaseAuditedEntity but as we have deleted that, maybe version is no longer required?

Reply:
Also, it might be worth switching message processing on in dev and perhaps also preprod (not sure how that fits with the security concerns but it is not live) before merging this, as that will show us if the changes actually work :-)

Reply:
I am ok with this PR as making further changes is more complicated and has very little gain.

Reply:
Sorry, my bad. we need this but forgot it was in the Baseaudit. Need to move the annotation to individual entities now. Thanks